lib/components/AlbumScreen/preset_chip.dart

```diff
@@ -0,0 +1,153 @@
+import 'dart:math';
+import 'package:flutter/material.dart';
+import 'package:finamp/services/queue_service.dart';
+import 'package:get_it/get_it.dart';
+
+const _radius = Radius.circular(4);
+const _borderRadius = BorderRadius.all(_radius);
+const _height = 36.0;
+final _defaultBackgroundColour = Colors.white.withOpacity(0.1);
+final _queueService = GetIt.instance<QueueService>();
+
+class PresetChips extends StatefulWidget {
+  const PresetChips({
+    Key? key,
+    required this.type,
+    required this.values,
+    required this.activeValue,
+    this.onTap,
+    this.mainColour,
+    this.onPressed,
+  }) : super(key: key);
+
+  // for future preset types other than "speed"
+  final String type;
+
+  final List<double> values;
+  final double activeValue;
```

Comment:
Does this always use the active value? Isn't it more of a `defaultValue` or `initialValue`?

Reply:
Because of the rebuilds, it is always the active value, I think. The `nowActiveValue` was unnecessary though.